mitodl/ocw-to-hugo
# remove unnecessary escape calls

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-to-hugo/issues/222
Closes https://github.com/mitodl/ocw-to-hugo/issues/221

#### What's this PR do?
This PR removes some code in the `baseurlshortcode` turndown rule that was taking markdown content and running it through `turndownService.escape` before returning.  It was causing markdown formatting characters in links to be pre-escaped before being written into the markdown file.  I believe this was originally done as part of [this](https://github.com/mitodl/ocw-to-hugo/pull/59) PR, but it doesn't seem to be necessary as I tested the course that was having the issue tied to it and the link in question still renders properly.

#### How should this be manually tested?
This change affects a large amount of content, because it's part of the turndown rule that processes all links containing the `baseurl` shortcode.  The best way to verify it is to s3 sync the entire `open-learning-course-data-production` bucket to your local machine, then run the following replacing my path to `open-learning -course-data` with your own:
```
mkdir private/test
node . -i ~/Code/open-learning-course-data -o private/test --rm
```
When you search the output folder at `private/test` for `[\*\*` there should only be results in 2 files, `private/test/6-441-information-theory-spring-2010/content/sections/readings.md` and `private/test/sts-464-cultural-history-of-technology-spring-2005/content/sections/readings.md`.  In both of these cases, the escaped double asterisk seems intentional.

I would also look at the 2 issues that this PR closes and inspect the output of the courses mentioned in those specifically, ensuring that markdown inside of link text is not escaped.  I would also look at the course mentioned in [this](https://github.com/mitodl/hugo-course-publisher/issues/166) issue and make sure there isn't a regression.
